api/server/container.go

```diff
@@ -94,7 +94,7 @@ func (s *Server) getContainersLogs(ctx context.Context, w http.ResponseWriter, r
 		return fmt.Errorf("Missing parameter")
 	}
 
-	// Validate args here, because we can't return not StatusOK after job.Run() call
+	// Validate args here, because we can't return not StatusOK after stream starts
```

Comment:
While you are at it, can you clarify this comment a it? `can't return not StatusOK` is super confusing.